lib/client/layout.js

```diff
@@ -114,10 +132,10 @@ BlazeLayout._render = function _render(layout, regions) {
     currentData = BlazeLayout._regionsToData(regions);
 
     currentLayout = Blaze._TemplateWith(currentData, function() {
-      return Spacebars.include(Template[layout]);
+      return Spacebars.include(BlazeLayout._getTemplate(layout, rootDomNode));
     });
 
-    Blaze.render(currentLayout, rootDomNode);
```

Comment:
The above change sets `RootComponent` instance as a `parentView` to layout template.